core/src/main/scala/org/apache/spark/sql/SnappySession.scala

```diff
@@ -1831,6 +1835,8 @@ object SnappySession extends Logging {
   private[this] val ID = new AtomicInteger(0)
   private[sql] val ExecutionKey = "EXECUTION"
 
+  def tokenizationDisabled: Boolean = java.lang.Boolean.getBoolean("DISABLE_TOKENIZATION")
```

Comment:
Same as above. Also this name can be changed to be more in-line with other names (e.g. snappydata.sql.tokenization). Isn't it better to make this a conf property instead of a system property?

Reply:
This is an existing name which I just changed the place from parser to snappysession object. Will do the needful.